website/blog/2020-10/00/_index.md

```diff
@@ -0,0 +1,111 @@
+
+This is the list of components:
+
+**(Virtual) Garden Cluster**
+- Gardener Extension API server
+- Validation Provider Webhooks
+- Project Namespace
+
+**Seed Cluster**
+- Gardenlet
+- Seed API server 
+  - every Shoot Control Plane has a dedicated namespace in the Seed.
+- Cloud Provider (owned by Stakeholder). 
+  - Arguably part of the Shoot cluster but used by components in the Seed cluster to create the infrastructure for the Shoot.
+- [Gardener DNS extension](https://github.com/gardener/external-dns-management)
+- Provider Extension (such as [gardener-extension-provider-aws](https://github.com/gardener/gardener-extension-provider-aws))
+- [Gardener Extension ETCD Druid](https://github.com/gardener/etcd-druid)
+- [Gardener Resource Manager](https://github.com/gardener/gardener-resource-manager)
+- Operating System Extension (such as [gardener-extension-os-gardenlinux](https://github.com/gardener/gardener-extension-os-gardenlinux))
+- Networking extension (such as [gardener-extension-networking-cilium](https://github.com/gardener/gardener-extension-networking-cilium))
+- [Machine Controller Manager](https://github.com/gardener/machine-controller-manager)
+- ContainerRuntime Extension (such as [gardener-extension-runtime-gvisor](https://github.com/gardener/gardener-extension-runtime-gvisor))
+- Shoot API server (in the Shoot Namespace in the Seed cluster)
+
+**Shoot Cluster**
+- Cloud Provider compute API (owned by Stakeholder) - for VM/Node creation
+- VM / Bare metal node hosted by Cloud Provider (in Stakeholder owned account)
+
+
+### How to use the diagram
+The diagram 
+- should be read from top to bottom - starting in the top left corner with the creation of the Shoot resource via the Gardener Extension API server.
+- should not require an encompassing documentation / description.
+  More detailed documentation on the components itself, can usually be found in the respective repository.
+- does not show which activities execute in parallel (many) and also does not describe the exact dependencies between the steps. 
+  This can be found out by [looking at the source code](https://github.com/gardener/gardener/blob/master/pkg/gardenlet/controller/shoot/shoot_control_reconcile.go).
+  It however tries to put the activities in a logical order of executing during the reconciliation flow. 
+
+Occasionally, there is an info box with additional information next to parts in the diagram that in my point of view require further explanation.
+Large example resource for the Gardener CRDs (e.g Worker CRD, Infrastructure CRD) are placed on the left side and are referenced by a dotted line (-----).
+
+Be aware, that Gardener is an evolving project, so the diagram will most likely be already outdated by the time you are reading this.
+Nevertheless, it should give a solid starting point for further explorations into the details of Gardener.
```

Comment:
Would it make sense to maintain it in the gardener repo and benefit from mutual effort to keep it up-to-date?

Reply:
My hope had been that it could become such a thing or referenced in the documentation even. 
I do not think that this will be the case in short term. 
I already asked if people want to contribute/check it, but people are mostly busy. 

Maybe if more people find this useful (externals too) we can strive to add it as an additional repository. 
But for now I would not want to create a repo in the official organization that will probably eventually be abandoned.

Reply:
There are other examples with a single maintainer too. I don't think that per-se is an issue and still leaves the door open for other contributors. But I leave this to you to decide.

Reply:
Talked to the colleagues and we think it makes most sense to add it into the documentation repository itself. Will adjust the PR.

Reply:
Agreed with @g-pavlov to leave it in the external repo.